### ISSUE TITLE
PositionalAssembler Error

### ISSUE THREAD

**Issue by DarioS** [PapenfussLab/gridss#260]
I have processed six TCGA patients seemingly successfully, but there's one which ran until the allotted time and didn't complete. In the comprehensive log, I notice

```
WARNING	2019-10-05 02:23:34	PositionalAssembler	Error during assembly of chromosome GL000216.1 (3080 reads in graph). Attempting recovery by rebuilding assembly graph.
java.lang.IllegalArgumentException: support and aes sizes do not match
	at au.edu.wehi.idsv.AssemblyAttributes.annotateAssembly(AssemblyAttributes.java:146)
	at au.edu.wehi.idsv.AssemblyFactory.createAssemblySAMRecord(AssemblyFactory.java:202)
	at au.edu.wehi.idsv.AssemblyFactory.createAnchoredBreakend(AssemblyFactory.java:41)
```

The log file is many times larger than the same type of file for the other patients and this error appears multiple times in the log file, about 1 minute apart. How can I resolve the mismatch of "support" and "aes"? Should I try making a minimal example to trigger this issue?

**Comment by d-cameron** [comment]
Gridss 2.6.2 or 2.6.1?

**Comment by DarioS** [comment]
It's version 2.6.1 I don't notice anything in the change log for version 2.6.2 that would seem to make a difference.

**Comment by d-cameron** [comment]
There is assembly recovery logic in there. If assembly dies for any reason,
2.6.2 will do a clean reload of the reads in the assembly graph so either
the edge case error will be avoided, or at least there will be a minimal
set of reads to reproduce the error (the lingering issues have been because
every time I try to reproduce locally, it works fine).

**Comment by DarioS** [comment]
The error recurs in version 2.6.2 Which file has the minimal set of reads which I should share?

**Comment by d-cameron** [comment]
Give me a few days and I'll add code to automatically package up the
minimal set of information required to reproduce any assembly error. Being
able to ask for _gridss_error_minimal_reproduction_data_.zip to be uploaded
with the issue should make finally fixing these remaining edge case bugs
much much more straight-forward.

**Comment by DarioS** [comment]
The problem seems fixed in version 2.6.3 of GRIDSS.

```
Wed Oct 23 05:27:54 AEDT 2019   Run complete with 19 warnings and 0 errors.
```
